Lib/platform.py

```diff
@@ -1230,6 +1230,59 @@ def platform(aliased=0, terse=0):
     _platform_cache[(aliased, terse)] = platform
     return platform
 
+### freedesktop.org os-release standard
+# https://www.freedesktop.org/software/systemd/man/os-release.html
+
+# NAME=value with optional quotes (' or ")
+_osrelease_line = re.compile(
+    "^(?P<name>[a-zA-Z0-9_]+)=(?P<quote>[\"\']?)(?P<value>.*)(?P=quote)$"
+)
+# /etc takes precedence over /usr/lib
+_osrelease_candidates = ("/etc/os-release", "/usr/lib/os-release", )
+_osrelease_cache = None
+
+
+def _parse_osrelease(lines):
+    # NAME and ID fields are mandatory fields with well-known defaults
+    # in pratice all Linux distributions override NAME and ID.
+    info = {
+        "NAME": "Linux",
+        "ID": "linux"
```

Comment:
You forgot:

```
"PRETTY_NAME": "Linux",
```

https://www.freedesktop.org/software/systemd/man/os-release.html